pkgs/racket-test/tests/racket/stress/parallel-custodian.rkt

```diff
@@ -0,0 +1,29 @@
+#lang racket/base
+
+(for ([i 10000])
+  (parallel-thread-pool-close (make-parallel-thread-pool 1)))
+
+(for ([i 10000])
+  (parameterize ([current-custodian (make-custodian)])
+    (make-parallel-thread-pool 1)
+    (custodian-shutdown-all (current-custodian))))
+
+(for ([i (in-range 1000)])
+  (define ts
+    (for/list ([j (in-range 8)])
+      (parameterize ([current-custodian (make-custodian)])
+        (thread
+         (lambda ()
+           (custodian-shutdown-all (current-custodian)))
+         #:pool 'own))))
+  (map thread-wait ts))
```

Comment:
Curious: why is this a `map` rather than `for-each`?

Reply:
A `for-each` would certainly make sense, but my fingers just happen to type `map` more readily.